### ISSUE TITLE
Questions about reproducing results

### ISSUE THREAD

**Issue by humanbins** [facebookresearch/deep_bisim4control#18]
Hi, thanks for providing the code of the paper. I have tried to reproduce the result, but get some trouble.
I want to reproduce the result of the default settings in DMC envs, such as walker_walk. 
However, it seems weird that my results seem bad. Since the default setting is without distractors, so I remove `--resource_files` and `--img_source` in the run_local.sh file:
```
DOMAIN=walker

TASK=walk
SEED=1
SAVEDIR=./save

MUJOCO_GL="egl" LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libGLEW.so:/usr/lib/nvidia-430/libGL.so.1.7.0  CUDA_VISIBLE_DEVICES=0 nohup python -u train.py \
    --domain_name ${DOMAIN} \
    --task_name ${TASK} \
    --agent 'bisim' \
    --init_steps 1000 \
    --num_train_steps 500000 \
    --encoder_type pixel \
    --decoder_type pixel \
    --transition_model_type 'ensemble' \
    --action_repeat 2 \
    --critic_tau 0.01 \
    --encoder_tau 0.05 \
    --decoder_weight_lambda 0.0000001 \
    --hidden_dim 1024 \
    --total_frames 1000 \
    --num_layers 4 \
    --num_filters 32 \
    --batch_size 128 \
    --init_temperature 0.1 \
    --alpha_lr 1e-4 \
    --alpha_beta 0.5 \
    --work_dir ${SAVEDIR}/${DOMAIN}_${TASK}_${SEED} \
    --save_tb \
    --seed ${SEED} $@ > ${DOMAIN}_${TASK}_${SEED}.log &`
```
I wonder if I have set some wrong hyperparameters? Or did I miss something important?

Here's some result:
![walker_walk](https://user-images.githubusercontent.com/22886818/126936665-1c8163c3-bc0d-417f-b707-4a4de21a2aff.png)

**Comment by amyzhang** [comment]
hi! you should use the hyperparameters set in the run_cluster.sh file.

**Comment by humanbins** [comment]
Thanks for the fast reply! I have checked the run_cluster.sh file, and I find that the batch_size is 512. I will try it again. Besides, should I reserve the `--IMG_SOURCE` and `--resource_files` as run_cluster.sh suggested even if I want to test in the default environments? Or I can ignore these two things?

**Comment by amyzhang** [comment]
you can ignore those, the default values should correspond to the default env with no distractors.

**Comment by humanbins** [comment]
Hi, Amy
I have tried to change the batch_size to 512. And test it with setting seed as `1`, the performance seems still worse. At 130K step, the batch_reward is about 0.13, and the reward is about 100-150.  I have checked some details, but still have no idea why this happens. Can the different setting of `MUJOCO_GL` affect the performance? It seems osmesa will cause some error in my server, so I changed to egl.

**Comment by DynaHMRC** [comment]
> Hi, Amy I have tried to change the batch_size to 512. And test it with setting seed as `1`, the performance seems still bad. At 130K step, the batch_reward is about 0.13, and the reward is about 100-150. I have checked some details, but still have no idea why this happens. Can the different setting of `MUJOCO_GL` affect the performance? It seems osmesa will cause some error in my server, so I changed to egl.

Hi, humanbins
Have you solved this problem now？Could you please share your experience?Thank you!
